src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,117 @@
+.cards-container {
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.card {
+
+  width: 200px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid var(--card-border);
+  border-radius: 5px;
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
```

Comment:
Same here, this folder is called cards.scss, and its the container of the single card. So you can put it in a separate file.